android/gradle.properties

```diff
@@ -18,3 +18,5 @@
 # org.gradle.parallel=true
 
 android.useDeprecatedNdk=true
+android.enableAapt2=false
```

Comment:
@william2958 I think you can remove this form the file. I am also not sure if it should be commited @seland?

Reply:
This change was in the setup-guide for Detox. It's part of the config